tools/adventure-pack/src/__tests__/iteratorPrototypeFilter.test.ts

```diff
@@ -0,0 +1,64 @@
+import { describe, expect, it } from "@jest/globals";
```

Comment:
Thanks for the PR! While you're here, add your name to the contributors list in `package.json`!